README.md

```diff
@@ -334,3 +334,17 @@ np
 ```
 The utility will ask you for the prefered version raise and if you are lucky, everything will run just fine.
 Finally, edit GitHub release form.
+
+### Preparing a package to upload to CamScripter
+If you  want to create your own package and upload it to CamScripter, you can use script CreatePackage. It creates zip file which contains all required files and directories in your package folder. The script accepts source code written either in JavaScript or TypeScript if the package has correct structure (more information in https://github.com/CamStreamer/CamScripterApp_examples/#readme). To include this script into your package add in file `package.json` following lines:
```

Comment:
If you want to create your own package and upload it to CamScripter App, you can use the script CreatePackage. It creates a zip file which contains all required files and directories in your package folder. The script accepts source code written either in JavaScript or TypeScript if the package has the correct structure (more information in https://github.com/CamStreamer/CamScripterApp_examples/#readme). To include this script in your package add the following lines in the file `package.json`:

Reply:
!